server/app/models/grid_domain_authorization.rb

```diff
@@ -9,6 +9,9 @@ class GridDomainAuthorization
   field :domain, type: String
   field :challenge, type: Hash
   field :challenge_opts, type: Hash # TODO encrypt?
+  field :authorization_type, type: String
+
+  has_one :grid_service
```

Comment:
It is set: https://github.com/kontena/kontena/pull/2565/files#diff-075d4bb6ae3e06ed6cca0d0d823de622R54 or in:
https://github.com/kontena/kontena/pull/2565/files#diff-075d4bb6ae3e06ed6cca0d0d823de622R56